packages/playwright-core/src/server/deviceDescriptors.ts

```diff
@@ -15,7 +15,8 @@
  * limitations under the License.
  */
 
-/**
- * @type {import('./types').Devices}
- */
-module.exports = require("./deviceDescriptorsSource.json")
+import type { Devices } from './types';
+
+import deviceDescriptorsSource from './deviceDescriptorsSource.json';
+
+export const deviceDescriptors = deviceDescriptorsSource as Devices;
```

Comment:
I am not against this change, but I wonder what's the reason?

Reply:
It makes imports more uniform, and besides it helps me in my https://github.com/ruifigueira/playwright-crx project, which uses vite to compile playwright to work on a chrome extension service worker, and vite doesn't like `requires` that much